osm2streets/src/operations/collapse_short_road.rs

```diff
@@ -51,34 +45,15 @@ impl StreetNetwork {
             let mut trim_roads_for_merging = BTreeMap::new();
             for i in [keep_i, destroy_i] {
                 for road in self.roads_per_intersection(i) {
-                    // If we keep this in there, it might accidentally overwrite the
```

Comment:
This comment doesn't make any sense. I think it was pre-opaque IDs, when deleted road borrowed IDs in a funny way